packages/ui/src/components/Organisms/ContentHub.tsx

```diff
@@ -99,3 +69,42 @@ export function ContentHub({ pageSize = 10 }: { pageSize: number }) {
     </div>
   );
 }
+
+export const Card = (item: ContentHubResultItemFragment) => (
```

Comment:
Nitpick: We don't need to `export` this, as it's only used in this file.

I would either remove `export` or move the `Card` function to its own component file (e.g. as a Molecule)